lib/constants/system_accounts.dart

```diff
@@ -1,26 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:seeds/datasource/remote/model/member_model.dart';
 import 'package:seeds/domain-shared/user_citizenship_status.dart';
-import 'package:seeds/i18n/constans/constans.i18n.dart';
 
 // Seeds system accounts with special icons. Harvest account, onboarding account, etc.
 
 // TODO(n13): get correct icons from figma
 class SystemAccounts {
   static final onboardingContract = MemberModel(
     account: 'join.seeds',
-    nickname: 'Onboarding Contract'.i18n,
+    nickname: '',
```

Comment:
Is this field (nikname) still necessary as empty string?

Reply:
Nope, `.nickname` is not used where MemberModel is a special system account. In that case `getLocalizedDisplayNameForSystemAccount(..)` is used 🤓 I'll add a comment.